docs/api/api-specification.md

```diff
@@ -1774,6 +1955,64 @@ On success, the server will update the existing resource given the id.
   </div>
 </details>
 
+<a name="update-put-immunization"/>
+
+### PUT `[base]/Immunization/[:id]`
+
+**Request Body:**
+<details>
+  <summary>Click to expand JSON snippet</summary>
+  <div markdown="1">
+
+```json
+{
+  "id": 32,
+  "meta": {
+    "lastUpdated": "2021-04-01T22:09:11+00:00"
+  },
```

Comment:
Remove meta from update example request body

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/797/commits/0c7bfdb96f9a95bcba17e573c31942f966608f4a.